config/crd/bases/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
Need to look at how to avoid creating this CRD in lvmo. `TestAPI` test fails if without the CRD currently.  Need to work around that. 
``` "topolvm volume snapshot class reconcile failure","Request.Name":"lvmcluster-sample","Request.Namespace":"lvm-operator-system","name":"odf-lvm-vg1","error":"volumesnapshotclasses.snapshot.storage.k8s.io is forbidden: User \"system:serviceaccount:lvm-operator-system:controller-manager\" cannot create resource \"volumesnapshotclasses\" in API group \"snapshot.storage.k8s.io\" at the cluster scope","stacktrace":"github.com/red-hat-storage/lvm-operator/controllers.(*LVMClusterReconciler).reconcile\n\t/workspace/controllers/lvmcluster_controller.go:180\ngithub.com/red-hat-storage/lvm-operator/controllers.(*LVMClusterReconciler).Reconcile\n\t/workspace/controllers ``` 

One suggestion from Nithya  was to have a `volumesnapshotEnabled` flag which is turned off in case of running the tests.